pkgs/tools/admin/certbot/0.10.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, python2Packages, fetchFromGitHub, dialog }:
+
+# Latest version of certbot supports python3 and python3 version of pythondialog
+
+python2Packages.buildPythonApplication rec {
+  name = "certbot-${version}";
+  version = "0.10.2";
+
+  src = fetchFromGitHub {
+    owner = "certbot";
+    repo = "certbot";
+    rev = "v${version}";
+    sha256 = "0j3i6381nb7x6vysi12swvry31bpzj8lg7ikkb5c50bciwbyvwxp";
+  };
+
+  propagatedBuildInputs = with python2Packages; [
+    ConfigArgParse
+    acme
+    configobj
+    cryptography
+    parsedatetime
+    psutil
+    pyRFC3339
+    pyopenssl
+    python2-pythondialog
+    pytz
+    six
+    zope_component
+    zope_interface
+  ];
+  buildInputs = [ dialog ] ++ (with python2Packages; [ nose mock gnureadline ]);
+
+  patchPhase = ''
+    substituteInPlace certbot/notify.py --replace "/usr/sbin/sendmail" "/var/setuid-wrappers/sendmail"
+    substituteInPlace certbot/util.py --replace "sw_vers" "/usr/bin/sw_vers"
+  '';
+
+  postInstall = ''
+    for i in $out/bin/*; do
+      wrapProgram "$i" --prefix PYTHONPATH : "$PYTHONPATH" \
```

Comment:
`buildPythonApplication` already creates wrappers that set `PYTHONPATH` in the `fixupPhase`, so there shouldn't be any need for including PYTHONPATH here.

Furthermore, never use `--prefix PYTHONPATH` because this can break in case you have a `PYTHONPATH` set in your environment.